_data/index.json

```diff
@@ -42,6 +42,10 @@
       {
         "name": "Phil Leggetter",
         "talk": "Pusher and real-time messaging"
+      },
+      {
+        "name": "Chris Martin"
+        "talk": "A five finger introduction to software engineering"
```

Comment:
Fixed in https://github.com/techmeetup/techmeetup.co.uk/commit/fa188f8ef761760abc0268a2a89446d8df43e3ff